xds/src/main/java/io/grpc/xds/Bootstrapper.java

```diff
@@ -76,12 +76,16 @@ BootstrapInfo readBootstrap() throws Exception {
                       StandardCharsets.UTF_8));
             } catch (Exception e) {
               failToBootstrapException = e;
+              throw e;
```

Comment:
Why add this line?

Reply:
Because I added this message in the wrapper exception:
`Failed to read bootstrap. A previous attempt has failed. See the cause for the original failure`

which is a description for second or later failures. So for the first failure I just throw instead of wrap.

Reply:
You do not need to add the description. There will only be one read IO globally, we do not need to expose this implementation. As long as we see an exception of bootstrap failure, we know it's __the bootstrap__ fails.

Reply:
We know it's the bootstrap fails, but without the description it may be confusing when "a previous attempt has failed". The description is just to be more informative.

Reply:
There is no "previous attempt" such a thing. All the following call of `readBootstrap()` is just reusing the cached result. A failure is just __THE failure__. The added description is redundant and misleading.

Reply:
"previous attempt' is not accurate. Pervious call of `readBootstrap()` is what I meant. Anyway, removed this line and message.